default.xml

```diff
@@ -725,7 +727,9 @@
 -->
   <project path="system/security" name="LineageOS/android_system_security" groups="pdk" />
   <project path="system/sepolicy" name="android_system_sepolicy" groups="pdk" remote="halium" />
+<!-- java-only repo
```

Comment:
tzdatacheck is actually being used in init (it wasn't disabled in 7.1 so I kept it in my 8.1 work as well: https://github.com/Halium/android_system_core/blob/halium-8.1/rootdir/init.rc#L524). See my suggestions for this in https://github.com/Herrie82/android_system_timezone/commit/cb536d1f58a659d84db75ee4d6471477e6d64df6 where I removed the Java bits and kept the non-Java for tzdatacheck